File: src/main/scala/warmUp/Scala/AVeryBigSum.scala
/**
  * @author Ram Saran Vuppuluri
  *         Problem Statement
  * @see https://www.hackerrank.com/challenges/a-very-big-sum/problem
  */
package warmUp.Scala

import java.util.Scanner

object AVeryBigSum extends App {
  val scanner = new Scanner(System.in)

  val arraySize = scanner.nextInt()

  println(arraySize)

  var longSum: Long = 0
  for (i <- 0 to arraySize - 1) {
    longSum += scanner.nextLong()
  }

  println(longSum)
}

File: src/main/scala/warmUp/Java/AVeryBigSum.java
/**
 * @author Ram Saran Vuppuluri
 * Problem Statement
 * @see https://www.hackerrank.com/challenges/a-very-big-sum/problem
 */
package warmUp.Java;

import java.util.Scanner;

public class AVeryBigSum {

    public static void main(String... args) {
        Scanner sin = new Scanner(System.in);

        int arraySize = sin.nextInt();
        long sumArray = 0;
        for (int i = 0; i < arraySize; i++) {
            sumArray += sin.nextLong();
        }
        System.out.println(sumArray);
    }
}
